src/sentry/migrations/0286_backfill_alertrule_organization.py

```diff
@@ -10,7 +10,7 @@ def backfill_alertrule_organization(apps, schema_editor):
     AlertRule = apps.get_model("sentry", "AlertRule")
     Organization = apps.get_model("sentry", "Organization")
 
-    for alert_rule in RangeQuerySetWrapperWithProgressBar(AlertRule.objects_with_snapshots.all()):
+    for alert_rule in RangeQuerySetWrapperWithProgressBar(AlertRule.objects.all()):
```

Comment:
I'm guessing the `objects_with_snapshots` method won't exist in the future?

Reply:
I thought this was causing an error in the tests, but realised it was something more complex

Reply:
Could you explain a bit why this change is needed?

Reply:
It's not, I ended up rolling this back once I realised the issue. The actual issue was that we had migration tests creating models using the Django models rather than the migration models. So when I added a new field, these failed since the migration tests roll back the database back to before that field exists.